### ISSUE TITLE
Clicking on TODO in the popup to cycle states without removing

### ISSUE THREAD

**Issue by redactedscribe** [ahonn/logseq-plugin-todo#37]
I'd like it if it were possible to cycle between TODO/DOING or LATER/NOW via the plugin's popup. This would be in alignment with Logseq's behaviour and would allow for quicker starting and stopping of time-tracking. Since you've also included a checkbox to complete a TODO, I don't see the harm in limiting the TODO/DOING label to cycling states instead of going from TODO, DOING, {removed-from-list}.

I'm sure there are a lot of people already accustomed to how this plugin does things, so if the idea is sound, I think it should be an option rather than a change.

Thanks for a great plugin.

**Comment by ahonn** [comment]
Yes, you're right, labels that are TODO items should not be removed from the list.
Actually, I think this should be a change, so I'm not considering adding an option.
